manager/manager.go

```diff
@@ -995,21 +1034,23 @@ func (self *manager) watchForNewContainers(quit chan error) error {
 	go func() {
 		for {
 			select {
-			case event := <-eventsChannel:
+			case event := <-self.eventsChannel:
 				switch {
-				case event.EventType == container.SubcontainerAdd:
-					err = self.createContainer(event.Name)
-				case event.EventType == container.SubcontainerDelete:
+				case event.EventType == watcher.SubcontainerAdd:
+					err = self.createContainer(event.Name, event.WatchSource)
+				case event.EventType == watcher.SubcontainerDelete:
 					err = self.destroyContainer(event.Name)
+				case event.EventType == watcher.SubcontainerOverride:
+					err = self.overrideContainer(event.Name, event.WatchSource)
```

Comment:
I have some concerns around how override is handled, but it might make more sense once the rkt watcher is added. Can we just remove the override event & method from this PR, and I'll review it when it's actually used?